source/src/molecular-unfolding/utility/MoleculeParser.py

```diff
@@ -90,4 +90,4 @@ def save(self, version, path=None):
     @classmethod
     def load(cls, filename):
         with open(filename, "rb") as f:
-            return pickle.load(f)
+            return pickle.load(f) #nosec
```

Comment:
why excluding `pickle` and `pickle.load()`?

Reply:
bandit reports pickle as below warning:
`Issue: [B301:blacklist] Pickle and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue.`